controllers/members.js

```diff
@@ -10,7 +11,7 @@ const tasks = require('../models/tasks')
 
 const getMembers = async (req, res) => {
   try {
-    const allMembers = await fetchMembers()
+    const allMembers = await fetchUsers()
```

Comment:
What do you think `allMembers` is better or `users` or `allUsers` ?

Reply:
I would go with `allMembers` here as we are actually assigning isMember value true or false property to it.

Reply:
We won't need to change API contracts and UI, just need to change variable name 

   ```
const allUsers = await fetchUsers()
     return res.json({
      message: allUsers.length ? 'Members returned successfully!' : 'No member found',
      members: allUsers
    })
```

Reply:
I did understand the approach you had meant, I was not convinced enough with the change in variable names from allMembers to allUsers so far.

Reply:
@bhk31 , I might be wrong here and would definitely stand to be corrected

Reply:
@harshith-venkatesh We can go ahead with either of the way. I don't have any problem with your approach. We can have opinion of @swarajpure @sumitd94 @ankurnarkhede  

@swarajpure @sumitd94 @ankurnarkhede Let us know your opinion.

Reply:
It is actually a simple PR that is getting extended by a long time leading to prod issue of null data being observed.
I would rather make the changes to get this PR resolved as I am feeling bad looking at the idle member page being in that state